src/containers/Home/Home.tsx

```diff
@@ -10,7 +10,7 @@ export function Home() {
   const handleSubmit = () => {
     const postalCodeRegex = /[a-zA-Z][0-9][a-zA-Z](-| |)[0-9][a-zA-Z][0-9]/;
     if (postalCodeRegex.test(postalCode)) {
-      history.push(`/search/${postalCode}`);
+      history.push(`/search/${postalCode.toLowerCase().replace(" ", "")}`);
```

Comment:
I think that maybe this normalization is better done within the `PharmacyList` where the query param is being set in the API call? The problem is that users could still be hitting the search route directly and it would be nice to cover all those scenarios. Just a thought. Or perhaps it's better extracted to a utility and done in both areas to ensure consistency everywhere + consistent URLs that we're setting on the browser.

Reply:
Good call.  I've added the lowercase & space removal there as well.  I'm going to leave this here still, as it's lightweight and gives us consistent urls.

I'm not going to worry about abstracting to a utility for this - it's lightweight enough to be doing a few js string functions and the manipulation is slightly different for both (i.e. full string vs only first three chars).